tools/alpine/Dockerfile

```diff
@@ -16,6 +16,10 @@ RUN cat /tmp/packages.$(uname -m) >> /tmp/packages && \
 # add the tools for WireGuard, since the kernel module is now included, but from edge/testing
 RUN apk fetch --recursive -o /mirror/$(uname -m) -X http://dl-cdn.alpinelinux.org/alpine/edge/testing -U wireguard-tools
 
+# add OVMF for EFI boot (x86 only)
+RUN [ $(uname -m) = x86_64 ] && \
+    apk fetch --recursive -o /mirror/$(uname -m) -X http://dl-cdn.alpinelinux.org/alpine/edge/community -U ovmf
```

Comment:
looks like community is enabled by default:
```
docker run --rm -ti alpine:3.6 cat /etc/apk/repositories
http://dl-cdn.alpinelinux.org/alpine/v3.6/main
http://dl-cdn.alpinelinux.org/alpine/v3.6/community
```
and we don't explicitly disable it.

Should we?